grails-plugin-gsp/src/test/groovy/org/grails/web/taglib/CountryTagLibTests.groovy

```diff
@@ -14,39 +14,47 @@
  */
 package org.grails.web.taglib
 
+import grails.testing.web.taglib.TagLibUnitTest
 import org.grails.plugins.web.taglib.CountryTagLib
+import org.grails.plugins.web.taglib.FormTagLib
 import org.springframework.web.servlet.support.RequestContextUtils as RCU
+import spock.lang.Specification
 
-class CountryTagLibTests extends AbstractGrailsTagTests {
+class CountryTagLibTests extends Specification implements TagLibUnitTest<FormTagLib> {
 
-    void testFullCountryListWithSelection() {
+    def testFullCountryListWithSelection() {
+        when:
         def template = '<g:countrySelect name="foo" value="gbr" />'
 
         def result = applyTemplate(template, [:])
 
-        assertResultContains result, '<option value="gbr" selected="selected" >United Kingdom</option>'
+        then:
+        assert result.contains('<option value="gbr" selected="selected" >United Kingdom</option>')
```

Comment:
You shouldn't have the `assert` keyword in the `then` block. The `CountryTagLib.ISO3166_3.each` could be replaced with `CountryTagLib.ISO3166_3.every` so it will return a `boolean`